mesecons_mvps/init.lua

```diff
@@ -63,28 +63,31 @@ end
 function mesecon.mvps_get_stack(pos, dir, maximum, all_pull_sticky)
 	-- determine the number of nodes to be pushed
 	local nodes = {}
-	local frontiers = {pos}
-
-	while #frontiers > 0 do
-		local np = frontiers[1]
-		local nn = minetest.get_node(np)
-
-		if not node_replaceable(nn.name) then
+	local pos_set = {}
+	local frontiers = mesecon.fifo_queue.new()
+	frontiers:add(pos)
```

Comment:
```suggestion
	frontiers:add(vector.new(pos))
```
The returned `nodes` should not have a reference to an argument, in case `pos` gets altered after the call there might be weird errors.